portability-core/src/main/java/org/dataportabilityproject/serviceProviders/microsoft/MicrosoftServiceProvider.java

```diff
@@ -61,14 +58,21 @@
         .put(ServiceMode.EXPORT, ImmutableList.of(PortableDataType.CALENDAR, PortableDataType.MAIL))
         .build();
 
+    private final static Map<PortableDataType, Map<ServiceMode, List<String>>> DATA_TYPE_SCOPES =
+        ImmutableMap.<PortableDataType, Map<ServiceMode, List<String>>>builder()
```

Comment:
This could be an Map<PortableDataType, ArrayListMultimap<SeviceMode, String>>

Reply:
Done. How does this look?